crates/modelardb_server/tests/integration_test.rs

```diff
@@ -797,6 +835,63 @@ fn ingest_time_series_and_flush_data(
     test_context.flush_data_to_disk();
 }
 
+/// Generates a time series with one tag and ingests it into a model table. Then a query is created
+/// from `segment_query` that cannot be rewritten so it is executed on segments, executes both,
```

Comment:
This comment is a bit hard to understand when read the first time. Maybe make it clear that `executes both` refers to `segment_query` and the created query.

Reply:
The doc comment has been changes to an enumerate list of steps performed to make it more clear what the function does.